### ISSUE TITLE
about the loss in the pgd_attack

### ISSUE THREAD

**Issue by lith0613** [MadryLab/cifar10_challenge#9]
The loss from CW in line 36 of pgd_attack.py use a negative sign, but there is not a such sign in the original CW loss, looking forward to your help
```     
 loss = -tf.nn.relu(correct_logit - wrong_logit + 50)
```

**Comment by dtsip** [comment]
The original CW attack is stated as a _minimization_ problem. Since our PGD attack _maximizes_ the loss, we need to modify the CW loss to accommodate for that.
